Sources/VSM/StateContainer/StatePublishing.swift

```diff
@@ -12,5 +12,15 @@ public protocol StatePublishing<State> {
     associatedtype State
     
     /// Publishes the state changes on the main thread
+    @available(*, deprecated, renamed: "didSetPublisher", message: "It has been renamed to didSetPublisher and will be removed in a future version.")
```

Comment:
nit - just say "Renamed to didSetPublisher and will be removed in a future version."
"It has been" is ambiguous and passive tense.